adapters/ix/ix_test.go

```diff
@@ -117,6 +161,86 @@ func TestIxTimeout(t *testing.T) {
 	}
 }
 
+func TestIxTimeoutMultipleSlots(t *testing.T) {
```

Comment:
Have you looked into supporting the native openrtb2 PBS API? It extracts handling the actual http calls, so you don't have to worry if you are also properly handling timeouts and such. Also has a JSON driven test suite that simplifies creating and extending test cases.

Reply:
I don't think we looked at that.  Can we open another issue to refactor the tests to use that?